src/app/blog/[slug]/components/SharePost.tsx

```diff
@@ -0,0 +1,46 @@
+'use client'
+
+import { useEffect, useState } from 'react'
+
+import { clsx } from 'clsx'
+
+import { Icon } from '@/components/Icon'
+import { touchTarget } from '@/components/Social'
+
+import { getSharePostSocialIcons } from '../utils/getSharePostSocialIcons'
+
+type SharePostProps = {
+  title: string
+  postTitle: string
+}
+
+export function SharePost({ title, postTitle }: SharePostProps) {
+  const [currentUrl, setCurrentUrl] = useState('')
+
+  useEffect(() => {
+    setCurrentUrl(window.location.href)
+  }, [])
+
+  const socialMediaIcons = getSharePostSocialIcons(currentUrl, postTitle)
+
+  return (
+    <div className="space-y-3 font-bold">
+      <p className="text-brand-300">{title}</p>
+      <ul className={clsx('flex w-full gap-6', touchTarget.offsetClass)}>
+        {socialMediaIcons.map(({ icon, href, label }, key) => (
+          <li key={key}>
+            <a
+              href={href}
+              className="hover:text-brand-400"
```

Comment:
It's missing the touch target class no?

```suggestion
className={clsx('inline-block hover:text-brand-400', touchTarget.class)}
```

Reply:
Ah yes, I put it in the wrong place :)